### ISSUE TITLE
Error when use LabelMe open the images collected by the autghors

### ISSUE THREAD

**Issue by Zhaosoi** [dvlab-research/LISA#191]
I noted the authors wrote in GitHub: "In ReasonSeg, we have collected 1218 images (239 train, 200 val, and 779 test). The training and validation sets can be download from this link."So, I downloaded these images. However, when I used LabelMe to open the images, error occured: “请确认C:/Users/zhaosoi/Desktop/train/3588328 892066223bo.json是一个合法的标签文件。” How can I view these labelled images?

**Comment by connorye** [comment]
labelme seems just to open format with itself labelme format,you can just view in text editor or use data_processing.py to see the vis version

**Comment by Zhaosoi** [comment]
> labelme seems just to open format with itself labelme format,you can just view in text editor or use data_processing.py to see the vis version

Many thanks! I successfully visualized images using the data_processing.py @connorye

**Comment by Zhaosoi** [comment]
@connorye Hi. I have another question. In the dataset (https://drive.google.com/drive/folders/125mewyg5Ao6tZ3ZdJ-1-E3n04LGVELqy?usp=sharing) published by the authors, the "explanatory" file only contains the "train.json". **There is no need for "val.json" like "train.json" during training, right?** 
The authors wrote in the GitHub: "Besides, we leveraged GPT-3.5 for rephrasing instructions, so images in the training set may have more than one instructions (but fewer than six) in the "text" field. During training, users may randomly select one as the text query to obtain a better model." In the "train.json" in the "explanatory" file, only one question exists for an image. If I want to make my own dataset, do I also need to design a lot of questions and then choose one from them? Or can I just design one question?

**Comment by connorye** [comment]
@Zhaosoi  Yes,u can just design one question for your text-label pair
